src/test/java/org/cactoos/collection/NoNullsTest.java

```diff
@@ -38,42 +38,45 @@
  */
 public final class NoNullsTest {
 
-    /**
-     * A rule for handling an exception.
-     */
-    @Rule
-    public final ExpectedException exception = ExpectedException.none();
-
     @Test
     public void throwsErrorIfNullInToArray() {
-        this.exception.expect(IllegalStateException.class);
-        this.exception.expectMessage(
-            "Item #1 of #toArray() is NULL"
+        MatcherAssert.assertThat(
```

Comment:
@atapin please use `Assertion` instead of the static methods from `MatcherAssert`